hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java

```diff
@@ -39,6 +42,41 @@ public void setGravatar(final boolean gravatar) {
         this.gravatar = gravatar;
     }
 
+
+
+    /**
+     * Localization information
+     */
+    public static class Localization implements Serializable {
+        private static final long serialVersionUID = 1L;
+
+        /**
+         * Default localization
+         */
+        private String defaultLocal;
+
+        /**
+         *  List of available localizations
+         */
+        private List<String> availableLocals;
```

Comment:
You could also add the available local "en" here as well, since it is the default